packages/perseus/src/renderer.tsx

```diff
@@ -1717,46 +1717,42 @@ class Renderer extends React.Component<Props, State> {
     };
 
     /**
-     * Returns an object mapping from widget ID to perseus-style score.
-     * The keys of this object are the values of the array returned
-     * from `getWidgetIds`.
+     * Grades the content.
```

Comment:
**nit**: We've been asked to use "score" over "grade". :) 

EDIT: Oh, I see that was the original comment from below. Would you mind updating it still? 

```suggestion
     * Scores the content.
```